patches/tModLoader/Terraria/Social/Base/DeveloperMetadata.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Newtonsoft.Json;
+using Terraria.ModLoader.Core;
+using Terraria.ModLoader.UI.ModBrowser;
+using Terraria.Social.Steam;
+
+namespace Terraria.Social.Base;
+
+public struct ModVersionHash
+{
+	private string hash;
```

Comment:
We want to make sure that json serializes this as a string, and not as an object with a `hash` field `{ hash: "..." }`